capnpc-go/fileparts.go

```diff
@@ -140,6 +149,10 @@ func (i *imports) byName(name string) (spec importSpec, ok bool) {
 }
 
 func (i *imports) add(spec importSpec) (name string) {
+	if i.used == nil {
+		i.used = make(map[string]bool)
```

Comment:
Why do this here, as opposed to just putting it at the top of newImports?

Reply:
No particular reason.  I think this is an artifact from the refactoring where there was some lazy initialization going on.

I'll make the change.

Reply:
Fixed.